keel-web/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -90,10 +97,7 @@ class ApplicationService(
           }
         }
 
-        ArtifactVersionSummary(
-          version = version,
-          environments = artifactSummariesInEnvironments.toSet()
-        )
+        return@map versionToSummary(artifact, version, artifactSummariesInEnvironments.toSet())
```

Comment:
> return@map

holy guacamole this is some freaky stuff

Reply:
The other options is very `groovy` and you don't have to put `return@map` there. But, I like stating it explicitly because I'm bad at reading code and this helps me.

Reply:
had to go look it up on SO — the way all these JVM langs handle closures totally breaks my brain coming from functional languages like JS. I love reading Kotlin PRs it's always an adventure 👨‍🔬